superset_patchup/oauth.py

```diff
@@ -271,6 +271,12 @@ def oauth_user_info(self, provider, response=None):
         # above)
         email_base = app.config.get("PATCHUP_EMAIL_BASE")
 
+        if is_valid_provider(provider, "commcare"):
+            logging.debug("Getting user info from {}".format(provider))
+            user = self.appbuilder.sm.oauth_remotes[provider].get("api/v0.5/identity/", token=response).json()
+            logging.debug("user - {}".format(user))
+            return user
+
         if is_valid_provider(provider, "onadata"):
```

Comment:
I think we can remove all these providers that we won't use.

Reply:
Yes, can do that.